src/meta/src/hummock/manager/mod.rs

```diff
@@ -321,6 +322,23 @@ where
                 .map(|version_delta| (version_delta.id, version_delta))
                 .collect();
 
+        if let Some((_, last_version_delta)) = hummock_version_deltas.last_key_value() {
+            for (compaction_group_id, group_deltas) in last_version_delta.get_group_deltas() {
+                if group_deltas.get_group_deltas().iter().any(|group_delta| {
+                    matches!(
+                        group_delta.delta_type.as_ref().unwrap(),
+                        DeltaType::GroupDestroy(_)
+                    )
+                }) {
+                    remove_compaction_group_in_sst_stat(
+                        &self.metrics,
+                        *compaction_group_id,
+                        CompactionConfigBuilder::default().build().get_max_level() as usize + 1,
```

Comment:
`max_level` got from default CompactionConfig may be incorrect, if the group has used a different config. One way is to keep some group info in DeltaType::GroupDestroy(_).

Reply:
> `max_level` got from default CompactionConfig may be incorrect, if the group has used a different config. One way is to keep some group info in DeltaType::GroupDestroy(_).

we can replace this with 100, since remove labels that do not exist is OK

Reply:
> `max_level` got from default CompactionConfig may be incorrect, if the group has used a different config. One way is to keep some group info in DeltaType::GroupDestroy(_).

fixed